products/green_fast_track/models/intermediate/spatial/shadows/int_spatial__shadow_open_spaces.sql

```diff
@@ -22,6 +22,6 @@ SELECT
     'shadow_open_spaces' AS flag_id_field_name,
     variable_type,
     variable_id,
-    raw_geom,
-    ST_BUFFER(raw_geom, 200) AS buffer_geom
+    ST_MULTI(COALESCE(lot_geom, raw_geom)) AS raw_geom,
+    ST_MULTI(ST_BUFFER(COALESCE(lot_geom, raw_geom), 200)) AS buffer_geom
```

Comment:
this change fixes a bug we didn't know about: all POPs records which joined to a lot during `int_spatial__pops` (virtually all of them) were not making it to `int_spatial__all`

this is why the pilot projects seed file changed